encrypted-media/drm-keystatuses-multiple-sessions.html

```diff
@@ -39,11 +39,12 @@
         .then( function( contents ) {
 
             // Select a content item with multiple keys and initData
-            var contentitem = contents.filter( function( item ) { return item.keys.length > 1 && item.initDataType && !item.associatedInitData; } )[ 0 ];
+            var contentitem = contents.filter( function( item ) { return item.keys.length > 1 && item.initDataType && !item.associatedInitData; } )[ 0 ],
```

Comment:
The `var` keyword is removed from the next line, so it's syntactically correct, but if it's confusing I could change it.